.cirrus.yml

```diff
@@ -118,7 +116,7 @@ report_stageone_task:
     remove_sos_script: &remove_sos |
         if [ $(command -v apt) ]; then
             apt -y purge sosreport
-            apt update
+            apt update --allow-releaseinfo-change
```

Comment:
I don't think this is correct. You will pull in Debian 11 packages into Debian 10. You'll end up with a weird hybrid 10/11. I recommend you choose a base image that has `buster` in `/etc/apt/sources.list` rather than `stable`.

Reply:
Doesn't `update` just refresh package availability info? We're only installing `python3-pip` to in turn pull in `avocado`.

I can edit the images in the GCP project, but I was under the impression `update` would not actually install new versions.

Reply:
Correct, but you then have metadata info from Debian 11. Any `apt-get install` command after that will install the Debian 11 version of it. That's why you'll end up with a weird hybrid. It's kind of like using Fedora 34 but pointing `/etc/yum.repos.d` to Fedora 35 (without actually running a distro upgrade).

Reply:
Ack. I'll update the images in the background, and add a note for future image/foreman updates.